shared/src/main/kotlin/org/jetbrains/kotlin/konan/target/KonanTargetExtenstions.kt

```diff
@@ -4,4 +4,20 @@ fun KonanTarget.supportsCodeCoverage(): Boolean =
         this == KonanTarget.MINGW_X64 ||
         this == KonanTarget.LINUX_X64 ||
         this == KonanTarget.MACOS_X64 ||
-        this == KonanTarget.IOS_X64
\ No newline at end of file
+        this == KonanTarget.IOS_X64
+
+fun KonanTarget.supportsOptimizedAllocator(): Boolean =
```

Comment:
`supportsOptimizedAllocator` is a bit unclear. Maybe `supportsMimallocAllocatior`?

Reply:
I don't want to connect this with exact allocator, there is possibility that we change or add another one.

Reply:
The problem is that the set of supported targets is determined by allocator itself, not target. The set for `jemalloc` or any other allocator may be different. 
Another possible solution is to add enum of alternative allocators and its entry to `supportsOptimizedAllocator`.

Reply:
Another good thing is to describe why other targets are not supported.

Reply:
Yes, it's a good idea with enum, but now it would be strange to have enumeration with one allocator. I don't want to show name of allocator in compiler options. Now we have just std and opt. It seems that same names everywhere are better.